functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SkipTestExecutionFunctionalTest.java

```diff
@@ -66,6 +69,9 @@ public void should_disable_smart_testing_and_execute_no_tests_when_test_executio
                 .inMode(ORDERING)
             .enable();
 
+        project
+            .applyAsLocalChanges("Enable both surefire and failsafe plugin with sample integration test");
+
```

Comment:
to verify it correctly, there should be the "verify" goal used in the `run()` command - to execute also failsafe tests